README.md

```diff
@@ -36,6 +36,16 @@ A transform is an object with these properties:
 * `name` is a required string. This is the name that will be shown in the list.
 * `description` is an optional string. If present, it will be displayed
   underneath the name in the list.
+* `variables` is an optional object, or function returning an object.
+  If present, after the user selects this transform, they will be prompted to
+  enter values for these variables in another dialog.
+  If a function is given, it will be called with the same props as `onSelected`
+  (except the `variables` values) and must return an object.
```

Comment:
`variables` should be changed to `variableValues`

Reply:
good catch, thanks!